ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,7 +15,7 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return (self.schedule(action) as RACDisposable) as Disposable?
```

Comment:
I'm confused, I think that example is slightly different?

I updated the PR with your suggested implementation though :)